README.md

```diff
@@ -1,30 +1,30 @@
-# Lance: A Columnar Data Format for Computer Vision
+# Lance: Deep Learning with DuckDB and Arrow
 
-![CI](https://github.com/eto-ai/lance/actions/workflows/cpp.yml/badge.svg)
-[![Docs](https://img.shields.io/badge/docs-passing-brightgreen)](https://eto-ai.github.io/lance/)
+<center>SELECT predict(‘resnet’, image) FROM dataset</center>
 
-[![PyPi](https://img.shields.io/pypi/v/pylance)](https://pypi.org/project/pylance/)
-![Python versions](https://img.shields.io/pypi/pyversions/pylance)
+Lance is an Arrow compatible data format and DuckDB extension designed to help you explore, get insights and use machine-learning on your large-scale visual data, such as video, images, point clouds, audio natively.
```

Comment:
at this point, should we not mention "data format"?

The paragraph below mentioned that

```
Lance is powered by <i>Lance Format</i>,
```

Reply:
Ok, should it be:

Lance is an Arrow and DuckDB compatible extension ... 

?

Reply:
sgtm